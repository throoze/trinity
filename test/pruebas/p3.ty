"holamundo"
"hola mundo"
"hola   mundo"
" hola   mundo "
"    hola   mundo    "

"hola \" mundo"
"hola\"mundo"
"\""

"hola \\ mundo"
"hola   \\   mundo"
"hola\\mundo"
"\\"

"hola \n mundo"
"hola   \n   mundo"
"hola\nmundo"
"\n"

"\\\"\n"
" \\ \" \n "
"    \\    \"    \n    "

"hola" "mundo"
"hola""mundo"

hola "mundo"
hola"mundo"

"hola" mundo
"hola"mundo

ho"la mun"do
"ho"la mun"do"

"hola"#mundo
"ho#la"mundo

hola_mundo
HOLA_MUNDO
HOLA_mundo
_hola_mundo

42hola27mundo
42 hola 27 mundo

"hola@mundo"
hola@mundo
