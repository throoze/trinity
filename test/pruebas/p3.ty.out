Linea 1, columna 1: String Literal: "holamundo"
Linea 2, columna 1: String Literal: "hola mundo"
Linea 3, columna 1: String Literal: "hola   mundo"
Linea 4, columna 1: String Literal: " hola   mundo "
Linea 5, columna 1: String Literal: "    hola   mundo    "
Linea 7, columna 1: String Literal: "hola \" mundo"
Linea 8, columna 1: String Literal: "hola\"mundo"
Linea 9, columna 1: String Literal: "\""
Linea 11, columna 1: String Literal: "hola \\ mundo"
Linea 12, columna 1: String Literal: "hola   \\   mundo"
Linea 13, columna 1: String Literal: "hola\\mundo"
Linea 14, columna 1: String Literal: "\\"
Linea 16, columna 1: String Literal: "hola \n mundo"
Linea 17, columna 1: String Literal: "hola   \n   mundo"
Linea 18, columna 1: String Literal: "hola\nmundo"
Linea 19, columna 1: String Literal: "\n"
Linea 21, columna 1: String Literal: "\\\"\n"
Linea 22, columna 1: String Literal: " \\ \" \n "
Linea 23, columna 1: String Literal: "    \\    \"    \n    "
Linea 25, columna 1: String Literal: "hola"
Linea 25, columna 8: String Literal: "mundo"
Linea 26, columna 1: String Literal: "hola"
Linea 26, columna 7: String Literal: "mundo"
Linea 28, columna 1: Identificador: hola
Linea 28, columna 6: String Literal: "mundo"
Linea 29, columna 1: Identificador: hola
Linea 29, columna 5: String Literal: "mundo"
Linea 31, columna 1: String Literal: "hola"
Linea 31, columna 8: Identificador: mundo
Linea 32, columna 1: String Literal: "hola"
Linea 32, columna 7: Identificador: mundo
Linea 34, columna 1: Identificador: ho
Linea 34, columna 3: String Literal: "la mun"
Linea 34, columna 11: Palabra reservada: do
Linea 35, columna 1: String Literal: "ho"
Linea 35, columna 5: Identificador: la
Linea 35, columna 8: Identificador: mun
Linea 35, columna 11: String Literal: "do"
Linea 37, columna 1: String Literal: "hola"
Linea 38, columna 1: String Literal: "ho#la"
Linea 38, columna 8: Identificador: mundo
Linea 40, columna 1: Identificador: hola_mundo
Linea 41, columna 1: Identificador: HOLA_MUNDO
Linea 42, columna 1: Identificador: HOLA_mundo
Linea 43, columna 1: Caracter inesperado: : _
Linea 43, columna 2: Identificador: hola_mundo
Linea 45, columna 1: Literal Numerico: 42.0
Linea 45, columna 3: Identificador: hola27mundo
Linea 46, columna 1: Literal Numerico: 42.0
Linea 46, columna 4: Identificador: hola
Linea 46, columna 9: Literal Numerico: 27.0
Linea 46, columna 12: Identificador: mundo
Linea 48, columna 1: String Literal: "hola@mundo"
Linea 49, columna 1: Identificador: hola
Linea 49, columna 5: Caracter inesperado: : @
Linea 49, columna 6: Identificador: mundo
